javav2/example_code/cloudfront/Readme.md

```diff
@@ -4,67 +4,173 @@ This README discusses how to run and test the Java code examples for Amazon Clou
+Define the following values to run the JUnit tests:
 
+**distributionId** - The distribution id of an exisiting CloudFront distribution. Follow [these steps](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/GettingStarted.SimpleDistribution.html) to manually
+set up a simple CloudFront distribution using the AWS console if you don't already have a distribution. 
+Use the distribution id as the value of this property (something similar to `E1LK64FWG38EAW`)
+
+**publicKeyFileName** - The name of the public key file that you copied to `src/main/resources`. This file is the public
+key counterpart of the private key you specify with the `privateKeyFullPath` property (for example `cf_public_key.pem`)
+
+**privateKeyFullPath** - The full system path and file name of the private key used for signing (for example, `/Users/user/cf_keypair/cf_private_key.der`).
+See the developer guide for [creating a key pair](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-trusted-signers.html#private-content-creating-cloudfront-key-pairs).
 
 ### Command line
+
 To execute the JUnit tests from the command line, you can use the following command.
 
 		mvn test
 
-You will see output from the JUnit tests, as shown here.
-
-	[INFO] -------------------------------------------------------
-	[INFO]  T E S T S
-	[INFO] -------------------------------------------------------
-	[INFO] Running Cloudfront
-	Test 1 passed
-	Test 2 passed
-	...
-	Done!
-	[INFO] Results:
-	[INFO]
-	[INFO] Tests run: 9, Failures: 0, Errors: 0, Skipped: 0
-	[INFO]
-	INFO] --------------------------------------------
-	[INFO] BUILD SUCCESS
-	[INFO]--------------------------------------------
-	[INFO] Total time:  12.003 s
-	[INFO] Finished at: 2020-02-10T14:25:08-05:00
-	[INFO] --------------------------------------------
-
+This Maven command will run both tests. You will see output from the JUnit tests similar to the following.
+```
+[INFO]
+[INFO] --- maven-surefire-plugin:2.22.1:test (default-test) @ CloudFrontJ2 ---
+[INFO]
+[INFO] -------------------------------------------------------
+[INFO]  T E S T S
+[INFO] -------------------------------------------------------
+[INFO] Running CloudFrontTest
+Test 1 passed
+Test 2 passed
+Test 3 passed
+Function name is FunctionUploadedByJava7820550d-c5d1-4923-9f55-176a497b5ee4
+Function runtime is cloudfront-js-1.0
+Test 4 passed
+The Distribution ARN is arn:aws:cloudfront::228601887821:distribution/EHUCVLZEPS70R
+Test 5 passed
+Test 6 passed
+FunctionUploadedByJava7820550d-c5d1-4923-9f55-176a497b5ee4 was successfully deleted.
+Test 7 passed
+[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.569 s - in CloudFrontTest
+[INFO] Running com.example.cloudfront.CloudFrontSigningTest
+2022-11-30 13:24:20 [main] INFO  com.example.cloudfront.S3TestUtils:27 - [cf-test-bucket4b560a63-fb6a-4d3e-af08-7a8d4ba589c1] bucket created
```

Comment:
Here and anywhere else a bucket name appears, change the bucket name to: 
DOC-EXAMPLE-BUCKET